src/content/ja/2020/capabilities.md

```diff
@@ -0,0 +1,328 @@
+  )
+}}
+
+`persist()`APIは `estimate()`メソッドよりも呼び出される頻度が低い。このAPIを利用しているモバイルページは176ページに過ぎないのに対し、デスクトップのWebサイトは25ページである。デスクトップでの利用はこのように低いレベルにとどまっているようですが、モバイルデバイスでは明確な傾向は見られません。
+
+## 新しい通知API
+
+Push APIとNotifications APIの助けを借りて、Webアプリケーションは以前からプッシュメッセージを受信したり、通知バナーを表示したりすることができるようになっていました。しかし、いくつかの部分が欠けていました。これまではプッシュメッセージはサーバーを経由して送信しなければならず、オフラインでスケジュールを立てることができませんでした。また、システムにインストールされているWebアプリケーションでは、アイコンにバッジを表示することができませんでした。バッジと通知トリガーAPIは、両方のシナリオを可能にします。
+
+### バッジAPI
+
+いくつかのプラットフォームでは、アプリケーションのアイコンに開いているアクションの量を示すバッジを表示するのが一般的です。例えば、バッジは未読のメールや通知、完了すべきTo-Do項目の数を表示できます。[バッジAPI](https://web.dev/badging-api/) ([W3C非公式ドラフト](https://w3c.github.io/badging/))では、インストールされたWebアプリケーションのアイコンにこのようなバッジを表示できます。開発者は `navigator.setAppBadge()` を呼び出すことでバッジを設定できます。このメソッドはアプリケーションのバッジに表示する番号を指定します。ブラウザはユーザーのデバイスに最も近い表示を行います。番号が指定されない場合は、一般的なバッジが表示されます（例: macOSでは白い点）。`navigator.clearAppBadge()`を呼び出すと、再びバッジが削除される。バッジAPIは、メールクライアントやソーシャルメディアアプリ、メッセンジャーに最適です。Twitter PWAでは、バッジAPIを利用してアプリのバッジに未読通知の数を表示しています。
+
+{{ figure_markup(
+  image="badging_api.png",
+  alt="バッジAPIを使用したChromeでのページ読み込みの割合",
+  caption='ChromeでバッジAPIを使用したページロードの割合<br>(ソース。<a href="https://chromestatus.com/metrics/feature/timeline/popularity/2726">バッジセット</a>, <a href="https://chromestatus.com/metrics/feature/timeline/popularity/2727">バッジクリア</a>)',
+  description="この機能を使用してChromeでページが読み込まれる割合に基づく、バッジAPIの使用状況のグラフです。セットメソッドとクリアメソッドを比較しています。両メソッドの使用量は時間の経過とともに増加しており、一般的にはセットメソッドの方が呼び出される頻度は高くなっています。2020年10月には、両方のメソッドの使用率が急上昇しており、ピークはセットメソッドがページロードの0.025％、クリアが0.016％となっています。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1145004925&format=interactive",
+  sheets_gid="1154751352"
+  )
+}}
+
+2020年4月にはGoogle Chrome 81が新しいバッジAPIを提供し、7月にはMicrosoft Edge 84が続いた。ChromeがAPIを提供した後、利用件数は急増した。2020年10月、Google Chromeの全ページロードの0.025%で`setAppBadge()`メソッドが呼び出される。`clearAppBadge()` メソッドが呼び出される頻度は低く、ページロードの約0.016%の間に呼び出されています。
+
+### 通知トリガーAPI
+
+Push APIでは、ユーザーが通知を受け取るためにはオンラインである必要があります。ゲーム、リマインダーやTo-doアプリ、カレンダー、目覚まし時計など、一部のアプリケーションでは、ローカルで通知の対象日を決定してスケジュールを組むことも可能でした。この機能をサポートするために、Chromeチームは[通知トリガー](https://web.dev/notification-triggers/)（[の説明者](https://github.com/beverloo/notification-triggers/blob/master/README.md)と呼ばれる新しいAPIを使って実験を行っています。このAPIは`options`マップに`showTrigger`という新しいプロパティを追加し、サービスワーカーの登録時に`showNotification()`メソッドへ渡すことができます。このAPIは将来的にさまざまな種類のトリガーに対応できるように設計されていますが、現時点では時間ベースのトリガーのみが実装されています。特定の日時に基づいて通知をスケジューリングするために、開発者は`TimestampTrigger`の新しいインスタンスを作成し、ターゲットのタイムスタンプをそれに渡すことができる。
+
+```js
+registration.showNotification('Title', {
+  body: 'Message',
+  showTrigger: new TimestampTrigger(timestamp),
+});
+```
+
+{{ figure_markup(
+  image="notification_triggers_api.png",
+  alt="Notification Triggers APIを使用したChromeでのページロードの割合",
+  caption='Notification Triggers APIを使用したChromeでのページ読み込みの割合<br>（出典 <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">通知トリガー</a>）',
```

Comment:
Some hidden characters here breaking the linting:

```suggestion
  caption='Notification Triggers APIを使用したChromeでのページ読み込みの割合<br>（出典 <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">通知トリガー</a>）',
```

Reply:
Thanks, I fixed.